### PR TITLE
add e2e for cluster terminal

### PR DESCRIPTION
**Pull request by zreigz**
**What does this PR do / Why do we need it**: add e2e for cluster terminal

**Does this PR close any issues?**:
Fixes #

```release-note
NONE
```

### CODE REVIEW

File: pkg/test/e2e/utils/client.go [hunk 1]
```diff
@@ -1911,3 +1912,22 @@ func (r *TestClient) SetMonitoringMLARateLimits(clusterID, projectID string, rat
...
+
+func (r *TestClient) ConnectClusterTerminal(token, clusterID, projectID string) (*websocket.Conn, error) {
+	endpoint, err := APIEndpoint()
+	if err != nil {
+		return nil, err
+	}
+
+	endpoint = strings.Replace(endpoint, "http", "ws", 1)
+	endpoint = fmt.Sprintf("%s/api/v1/ws/projects/%s/clusters/%s/terminal", endpoint, projectID, clusterID)
+	fmt.Println(endpoint)
```
Comment (hbernardo) [inline pkg/test/e2e/utils/client.go]: Should this "print" still be here?

**Review by hbernardo — APPROVED** [review]
/lgtm
/approve

File: pkg/test/e2e/utils/client.go (hunk 1 above)
Comment (zreigz) [reply pkg/test/e2e/utils/client.go]: I will remove it, thanks

**Pull request merged**